### ISSUE TITLE
SetpointManager:MixedAir warning Reference Node is the same as the SetPoint Node should be fatal

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5631]
## Issue overview

User file reported system node setpoints on the order of -174C, because the SetpoinManager;Mixed air had this error:

```
** Warning ** GetSetPointManagerInputs: SetpointManager:MixedAir="MIXED AIR TEMP", reference node.
** ~~~ ** ..Reference Node is the same as the SetPoint Node
** ~~~ ** Reference Node Name="AIRLOOPSETPOINTNODE".
```

I cannot think of any situation where this would be valid.  In this case there was no other setpoint manager acting on this node, so the setpoint temperature just kept getting colder.  But even if there was another setpoint manager, it does not make sense to adjust the reference node setpoint.  This should be changed to a fatal error.
### Details

Some additional details for this issue (if relevant):
- Platform Win64
- Version of EnergyPlus 8.5.0
- Helpdesk ticket 11296.
### Checklist

Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
- [x] Defect file added (EnergyPlusDevSupport\DefectFiles)
- [x] Ticket added to Pivotal for defect (development team task)
- [x] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by rraustad** [comment]
Simple enough. Are you fixing?

**Comment by mjwitte** [comment]
No, feel free to take it.

**Comment by rraustad** [comment]
Program now fatals out.

```
** Severe  ** GetSetPointManagerInputs: SetpointManager:MixedAir="MIXED AIR TEMP", reference node.
**   ~~~   ** ..Reference Node is the same as the SetPoint Node
**   ~~~   ** Reference Node Name="AIRLOOPSETPOINTNODE".
**  Fatal  ** GetSetPointManagerInputs: Errors found in input.  Program terminates.
```

**Comment by rraustad** [comment]
See PR https://github.com/NREL/EnergyPlus/pull/5642

**Comment by Myoldmopar** [comment]
Fixed via #5642
